Jenkinsfile

```diff
@@ -34,8 +34,8 @@ pipeline {
         stage ('Update branches of required packages') {
             steps {
                 // When using the docker container, we need to change the HOME path
-                // to WORKSPACE to have the authority to install the packages.
-                withEnv(["HOME=${env.WORKSPACE}"]) {
```

Comment:
uhm.. I'd actually recommend doing something like:

```
withEnv(["WHOME=${env.WORKSPACE}"]) {
```

instead.. :( 

some steps end up using that information..